packages/@aws-cdk/aws-location-alpha/test/integ.tracker.ts

```diff
@@ -0,0 +1,37 @@
+import { App, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import * as kms from 'aws-cdk-lib/aws-kms';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import { PositionFiltering, Tracker } from '../lib/tracker';
+import { GeofenceCollection } from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    const geofenceCollection1 = new GeofenceCollection(this, 'GeofenceCollection1', {});
+    const geofenceCollection2 = new GeofenceCollection(this, 'GeofenceCollection2', {});
+    const geofenceCollectionForAdd1 = new GeofenceCollection(this, 'GeofenceCollectionForAdd1', {});
+    const geofenceCollectionForAdd2 = new GeofenceCollection(this, 'GeofenceCollectionForAdd2', {});
+
+    const key = new kms.Key(this, 'Key', { removalPolicy: RemovalPolicy.DESTROY });
+
+    const tracker = new Tracker(this, 'Tracker', {
+      trackerName: 'MyTracker',
+      description: 'test tracker',
+      eventBridgeEnabled: true,
+      kmsKeyEnableGeospatialQueries: true,
+      kmsKey: key,
+      positionFiltering: PositionFiltering.ACCURACY_BASED,
+      geofenceCollections: [geofenceCollection1, geofenceCollection2],
+    });
+
+    tracker.addGeofenceCollections(geofenceCollectionForAdd1, geofenceCollectionForAdd2);
+  }
+}
+
+const app = new App();
+
+new integ.IntegTest(app, 'TrackerTest', {
```

Comment:
Thanks for the PR @mazyu36  - I'm wondering if it's possible to add some integ test assertions here to validate that the construct works beyond deployment

Reply:
Thank you. I added an assertion to ensure that the tracker consumers are correctly created. 
How does this look?

Reply:
@mazyu36 where is the assertion? I don't see it in this file

Reply:
@sumupitchayan 
The following line. Please let me know if this is insufficient.

https://github.com/aws/aws-cdk/blob/a0d0cd39d68377e818256d5723d55bb75973cca8/packages/%40aws-cdk/aws-location-alpha/test/integ.tracker.ts#L52

Reply:
Ah yes sorry I missed this for some reason - thanks for adding this! Approved 👍